xml/snmp_sysdescr.xml

```diff
@@ -5965,6 +5965,26 @@ Copyright (c) 1995-2005 by Cisco Systems
     <param pos="1" name="os.product"/>
     <param pos="2" name="os.version"/>
   </fingerprint>
+  <!--======================================================================
+                            pfSense
+   =======================================================================-->
+  <fingerprint pattern="^pfSense\s(\S+)\s(\d+\.\d+(?:\.\d+)?)-RELEASE.*(FreeBSD)\s([\d\.]+-(?:STABLE|RELEASE)(?:-p\d+)?).*\s(\w+)$">
+    <description>pfSense</description>
+    <example service.vendor="FreeBSD" service.product="bsnmpd" service.component.vendor="pfSense" service.component.product="pfSense" service.component.version="2.3.4" os.vendor="FreeBSD" os.family="FreeBSD" os.product="FreeBSD" os.version="10.3-RELEASE-p19" os.arch="amd64" os.certainty="0.9" host.name="pfSense.localdomain">pfSense pfSense.localdomain 2.3.4-RELEASE pfSense FreeBSD 10.3-RELEASE-p19 amd64</example>
```

Comment:
FWIW, there is no need to test the values that you are not extracting from the provided data (in this case, `os.certainty`).